pal/src/thread/pal_thread.cpp

```diff
@@ -2834,8 +2834,28 @@ int CorUnix::CThreadMachExceptionHandlers::GetIndexOfHandler(exception_mask_t bm
 
 #endif // HAVE_MACH_EXCEPTIONS
 
+#ifndef __APPLE__
+#define THREAD_LOCAL thread_local
+#else
+#define THREAD_LOCAL _Thread_local
+#endif
+
+#ifndef __IOS__
+static THREAD_LOCAL ULONG_PTR s_cachedThreadStackHighLimit = 0;
+static THREAD_LOCAL ULONG_PTR highLimit_cache = 0;
+static THREAD_LOCAL ULONG_PTR lowLimit_cache = 0;
```

Comment:
nit: `highLimit_cache` and `s_cachedThreadStackHighLimit` seems to be the same thing duplicated.
nit: global static variable naming inconsistency.